src/UI/UIUtils.java

```diff
@@ -320,6 +320,11 @@ public static COSpriteSpec fromDisk(String col, String fac)
   public static final COSpriteSpec AR = COSpriteSpec.fromDisk("Acid"  , "Rain");
   public static final COSpriteSpec WN = COSpriteSpec.fromDisk("White" , "Nova");
 
+  public static final COSpriteSpec BW = COSpriteSpec.fromDisk("Brenner's"     , "Wolves");
+  public static final COSpriteSpec NW = COSpriteSpec.fromDisk("New Rubinelle" , "Wolves");
+  public static final COSpriteSpec LA = COSpriteSpec.fromDisk("Lazurian"      , "Army");
+  public static final COSpriteSpec IA = COSpriteSpec.fromDisk("IDS"           , "Army");
```

Comment:
Two wolves and two armies? Duplicate names isn't ideal, but I guess maybe it's hard to avoid. Also, wasn't Rubinelle also an army?

Reply:
It's an artifact of how we set up unit sprites.
In the lens of unit sprites, there are only two "real" armies in DoR: Rubinelle-based and Lazuria-based. IDS uses the Lazuria sprites.
Because of that, "Wolves" has the right infantry sprites for the NRA.